internetarchive/cli/ia_download.py

```diff
@@ -40,7 +40,8 @@
     -P, --search-parameters=<key:value>...   Download items returned from a specified search query.
     -g, --glob=<pattern>                     Only download files whose filename matches the
                                              given glob pattern.
-    -f, --format=<format>...                 Only download files of the specified format(s).
+    -f, --format=<format>...                 Only download files of the specified format.
+                                             Use this option multiple times to download multiple formats.
```

Comment:
```suggestion
                                             Use this option multiple times to download multiple
                                             formats.
```
Placate flake8 line limits...  https://travis-ci.org/github/jjjake/internetarchive/jobs/728058941#L231

Reply:
Thanks for pointing that out.  Fixed.